demos/SSHRemoting/README.md

```diff
@@ -1,20 +1,33 @@
 
 ## Overview
-PowerShell remoting normally uses WinRM for connection negotiation and data transport.  SSH was chosen for this remoting implementation since it is now available for both Linux and Windows platforms and allows true multiplatform PowerShell remoting.  However, WinRM also provides a robust hosting model for PowerShell remote sessions which this implementation does not yet do.  And this means that PowerShell remote endpoint configuration and JEA (Just Enough Administration) is not yet supported in this implementation.
+PowerShell remoting normally uses WinRM for connection negotiation and data transport.
+SSH was chosen for this remoting implementation since it is now available for both Linux and Windows platforms and allows true multiplatform PowerShell remoting.
+However, WinRM also provides a robust hosting model for PowerShell remote sessions which this implementation does not yet do.
+And this means that PowerShell remote endpoint configuration and JEA (Just Enough Administration) is not yet supported in this implementation.
 
-PowerShell SSH remoting lets you do basic PowerShell session remoting between Windows and Linux machines.  This is done by creating a PowerShell hosting process on the target machine as an SSH subsystem.  Eventually this will be changed to a more general hosting model similar to how WinRM works in order to support endpoint configuration and JEA.
+PowerShell SSH remoting lets you do basic PowerShell session remoting between Windows and Linux machines.
+This is done by creating a PowerShell hosting process on the target machine as an SSH subsystem.
+Eventually this will be changed to a more general hosting model similar to how WinRM works in order to support endpoint configuration and JEA.
 
 The New-PSSession, Enter-PSSession and Invoke-Command cmdlets now have a new parameter set to facilitate this new remoting connection
 ```powershell
-[-HostName <string>]  [-UserName <string>]  [-KeyPath <string>]
+[-HostName <string>]  [-UserName <string>]  [-KeyFilePath <string>]
 ```
 This new parameter set will likely change but for now allows you to create SSH PSSessions that you can interact with from the command line or invoke commands and scripts on.
-You specify the target machine with the HostName parameter and provide the user name with UserName.  When running the cmdlets interactively at the PowerShell command line you will be prompted for a password.  But you also have the option to use SSH key authentication and provide a private key file path with the KeyPath parameter.  Note that PSCredential is not yet supported.
+You specify the target machine with the HostName parameter and provide the user name with UserName.
+When running the cmdlets interactively at the PowerShell command line you will be prompted for a password.
+But you also have the option to use SSH key authentication and provide a private key file path with the KeyFilePath parameter.
+Note that PSCredential is not supported.
```

Comment:
I wonder if we should remove this and just not mention PSCredential.  The final SSH cmdlet parameter sets don't include PSCredential so there should be no confusion.